client/configuration.go

```diff
@@ -83,12 +87,26 @@ func (c Configuration) internal(basePath string) (*internal.Configuration, error
 
 	userAgent := fmt.Sprintf("bloxone-%s/%s", sdkIdentifier, version)
 
-	return &internal.Configuration{
+	ic := &internal.Configuration{
 		DefaultHeader:    defaultHeaders,
 		UserAgent:        userAgent,
 		Debug:            false,
 		OperationServers: nil,
 		Servers:          []internal.ServerConfiguration{{URL: cspURL}},
 		HTTPClient:       httpClient,
-	}, nil
+		DefaultTags:      make(map[string]string),
+	}
+	// Add default tags set
+	if c.DefaultTags != nil {
+		ic.AddDefaultTags(c.DefaultTags)
+	}
+
+	// setting up custom tag to identify the client
+	dfTags := make(map[string]string)
+	// Extract client from ClientName string
+	// Format: <client>/version#commit
+	dfTags[clientIdentifier] = strings.Split(c.ClientName, "/")[0]
+	ic.AddDefaultTags(dfTags)
+
```

Comment:
Should we leave this section disabled by default? We can add the enable flag through terraform provider configuration

Reply:
removed this section of the code, as we no longer implement this